### ISSUE TITLE
write tests for routes

### ISSUE THREAD

**Issue by alubling** [alubling/maskstore#14]
I'm actually using Postman to test every route to make sure it is successfully doing what it's supposed to. But once we build the Factories we can test that the entire route chain goes through as it's supposed to.

**Comment by bfrost12** [comment]
I'm actually going to try writing the tests for the backend express/server routes separately from the $http requests from the front end.  If it gets too complicated I'll revert to the original strategy.

**Comment by alubling** [comment]
good idea Bryce, and yes like the idea of writing the route tests separately.
